media/css/115release.css

```
@@ -0,0 +1,1178 @@
+  --color-green-70: #15803d;
+  --color-green-80: #166534;
+  --color-green-90: #14532d;
+
+  --color-teal-10: #cdfaf7;
+  --color-teal-20: #9ff4f0;
+  --color-teal-30: #62e9e6;
+  --color-teal-40: #27d3d6;
+  --color-teal-50: #0db7bd;
+  --color-teal-60: #0a929d;
+  --color-teal-70: #0e757f;
+  --color-teal-80: #135e67;
+  --color-teal-90: #144e56;
+
+  --color-blue-10: #ddeefe;
+  --color-blue-20: #bce0fd;
+  --color-blue-30: #88ccfc;
+  --color-blue-40: #4cb1f9;
+  --color-blue-50: #2493ef;
+  --color-blue-60: #1373d9;
+  --color-blue-70: #105bbc;
+  --color-blue-80: #124c9a;
+  --color-blue-90: #15427c;
+
+  --color-purple-10: #f3e8ff;
+  --color-purple-20: #e9d5ff;
+  --color-purple-30: #d8b4fe;
+  --color-purple-40: #c084fc;
+  --color-purple-50: #a855f7;
+  --color-purple-60: #9333ea;
+  --color-purple-70: #7e22ce;
+  --color-purple-80: #6b21a8;
+  --color-purple-90: #581c87;
+
+  --color-magenta-10: #fbe7f9;
+  --color-magenta-20: #f8cff3;
+  --color-magenta-30: #f4a9e8;
+  --color-magenta-40: #ee75d7;
+  --color-magenta-50: #e247c4;
+  --color-magenta-60: #cd26a5;
+  --color-magenta-70: #b01a86;
+  --color-magenta-80: #91186e;
+  --color-magenta-90: #79195c;
+
+  --color-brown-10: #f4e9d7;
+  --color-brown-20: #efdfc4;
+  --color-brown-30: #e4cdab;
+  --color-brown-40: #d7bc96;
+  --color-brown-50: #b6986c;
+  --color-brown-60: #96764b;
+  --color-brown-70: #755b38;
+  --color-brown-80: #51412c;
+  --color-brown-90: #47341f;
+
+  --color-gray-05: #fafafa;
+  --color-gray-10: #f4f4f5;
+  --color-gray-20: #e4e4e7;
+  --color-gray-30: #d4d4d8;
+  --color-gray-40: #a1a1aa;
+  --color-gray-50: #71717a;
+  --color-gray-60: #52525b;
+  --color-gray-70: #3f3f46;
+  --color-gray-80: #27272a;
+  --color-gray-90: #18181b;
+
+  --color-ink-10: #f1f3fa;
+  --color-ink-20: #e3e5f2;
+  --color-ink-30: #cdd0e5;
+  --color-ink-40: #9b9ec2;
+  --color-ink-50: #6e6f9b;
+  --color-ink-60: #52507c;
+  --color-ink-70: #3e3c67;
+  --color-ink-80: #2a284b;
+  --color-ink-90: #1a1838;
+
+  --color-white: #ffffff;
+  --color-black: #000000;
+
+  --bg: #31293e;
+  --txt: var(--color-gray-05);
+
+  --max-width: 1280px;
+  --slider: 50%;
+
+  --icon-glow: var(--color-gray-50);
+  --info-header: var(--color-gray-20);
+  --info-txt: var(--color-gray-10);
+
+  --button-background-color: var(--color-gray-20);
+  --button-hover-text-color: var(--color-gray-70);
+  --button-hover-background-color: var(--color-gray-10);
+  --button-active-background-color: var(--color-gray-30);
+  --button-border-color: var(--color-gray-40);
+  --button-border-radius: 6px;
+  --button-border-size: 2px;
+  --button-text-color: var(--color-gray-70);
+  --button-margin: 9px;
+  --button-padding: 9px;
+  --focus-outline-color: var(--color-blue-50);
+  --focus-outline-offset: 4px;
+  --focus-outline: 4px solid var(--focus-outline-color);
+  --icon-size: 32px;
+}
+
+body {
+  padding: 0;
+  margin: 0;
+  font-size: 1rem;
+  background-color: var(--bg);
+  color: var(--txt);
```

The header image should be on the body so it can scale properly like is in the mock-up
```suggestion
  color: var(--txt);
  background-image: url("/media/img/thunderbird/backgrounds/blob.png");
  background-repeat: no-repeat;
  background-position: center -300px;
  background-size: 80vw;
```